configman/config_manager.py

```diff
@@ -461,7 +461,10 @@ def write_conf(self, config_file_type, opener, skip_keys=None):
                 an_option = option_defs[a_key]
                 if ((not a_key.startswith('admin'))
                     and isinstance(an_option, Option)
-                    and an_option.secret
+                    and (
+                        an_option.secret
+                        or 'password' in an_option.name.lower()
+                    )
```

Comment:
it wouldn't be difficult.  The word password appears in the socorro code 87 times.  Only the uses where a configman Option is created are important.